src/widgets/allergies/allergy-record.component.tsx

```diff
@@ -31,137 +32,123 @@ export default function AllergyRecord(props: AllergyRecordProps) {
-      >
-        <div
-          className={`omrs-type-body-regular ${styles.allergyCard} ${
-            allergy.severity.display === "Severe"
-              ? `${styles.highSeverity}`
-              : `${styles.lowSeverity}`
-          }`}
-        >
-          <div className={`omrs-type-title-3 ${styles.allergyName}`}>
-            <span data-testid="allergy-name">{allergy.display}</span>
-          </div>
-          <table className={styles.allergyTable}>
-            <thead className="omrs-type-body-regular">
-              <tr>
-                <th>Severity</th>
-                <th>Reaction</th>
-                <th>Onset Date</th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr>
-                <td data-testid="severity">
-                  <div
-                    className={`${styles.centerItems} ${
-                      allergy.severity.display === "Severe" ? `omrs-bold` : ``
-                    }`}
-                    style={{
-                      textTransform: "uppercase",
-                      letterSpacing: "1px",
-                      lineHeight: "1rem",
-                      fontWeight: 500
-                    }}
-                  >
-                    {allergy.severity.display === "Severe" && (
-                      <svg
-                        className={`omrs-icon`}
-                        fontSize={"15px"}
-                        fill="var(--omrs-color-danger)"
+  return (
+    <>
+      {!!(allergy && Object.entries(allergy)) && (
+        <div className="styles.allergyContainer">
+          <SummaryCard
+            name="Allergy"
+            styles={{ width: "100%" }}
+            editComponent={AllergyForm}
+            showComponent={() =>
+              openWorkspaceTab(AllergyForm, "Edit Allergy", {
+                allergyUuid: allergy.uuid
+              })
+            }
+            link={`/patient/${patientUuid}/chart/allergies`}
+          >
+            <div
+              className={`omrs-type-body-regular ${styles.allergyCard} ${
+                allergy.severity.display === "Severe"
+                  ? `${styles.highSeverity}`
+                  : `${styles.lowSeverity}`
+              }`}
+            >
+              <div className={`omrs-type-title-3 ${styles.allergyName}`}>
+                <span data-testid="allergy-name">{allergy.display}</span>
+              </div>
+              <table className={styles.allergyTable}>
+                <thead className="omrs-type-body-regular">
+                  <tr>
+                    <th>Severity</th>
+                    <th>Reaction</th>
+                    <th>Onset Date</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td data-testid="severity">
+                      <div
+                        className={`${styles.centerItems} ${
+                          allergy.severity.display === "Severe"
```

Comment:
Can we put the special identifier `Severe` in a common place such that it can be easily found and changed? Thanks!

Reply:
Is there a consensus on where this common place should be?

Reply:
I don't think so - you could place it in a file like "constants.ts" or something similar. Of course, since there is no consensus we could have it in the same module, too. The only thing I would try to avoid at this point is to have the constant "disappearing" in the logic.

Reply:
I've moved them to an enum for now.